### PR TITLE
Fix Plaid evn handling

### PR DESCRIPTION
**Pull request by andrewbuccilli**
- includes black formatting

### CODE REVIEW

File: app/constants.py [hunk 1]
```diff
@@ -10,19 +10,18 @@
...
+PLAID_ENV = environ.get("PLAID_ENV", "sandbox")
 
-# can be sandbox.dev, api.dev, sandbox(prod), api(prod), 
+# can be sandbox.dev, api.dev, sandbox(prod), api(prod),
 # api.staging, sandbox.staging.
-BOND_ENV = environ.get("BOND_ENV","sandbox")
+BOND_ENV = environ.get("BOND_ENV", "sandbox")
 
 plaid_host = f"https://{PLAID_ENV}.plaid.com"
-bond_host = f"https://{BOND_ENV}.dev.bond.tech"
```
Comment (andrewbuccilli) [inline app/constants.py:20]: @adityabond removed dev

File: app/constants.py [hunk 2]
```diff
@@ -10,19 +10,18 @@
...
+bond_host = f"https://{BOND_ENV}.bond.tech"
 
 api_docs = "https://docs.bond.tech/reference"
 
-PLAID_CLIENT_ID=environ.get("PLAID_CLIENT_ID",None)
-PLAID_SECRET=environ.get("PLAID_SECRET", None)
-PLAID_PRODUCTS=environ.get("PLAID_PRODUCTS","transactions")
-PLAID_COUNTRY_CODES=environ.get("PLAID_COUNTRY_CODES","US")
-PLAID_ENV=environ.get("PLAID_ENV", "sandbox")
```
Comment (andrewbuccilli) [inline app/constants.py:28]: @adityabond removed this line

File: app/handlers.py [hunk 3]
```diff
@@ -4,140 +4,139 @@
...
+            "Authorization": authorization,
+            "Content-type": "application/json",
+        }
+
+        r = requests.patch(url=url, headers=headers, json=payload)
+        return r.json()
 
 
 # createLinkToken -> initializePlaidLink -> createAccessToken
-def plaid_bond_test(account_id, PLAID_ENV):
```
Comment (andrewbuccilli) [inline app/handlers.py:139]: removed argument

File: app/handlers.py [hunk 4]
```diff
@@ -300,7 +298,7 @@ def plaid_bond_micro_deposit_test(account_id, linked_account_id):
           .then( data => {{
           console.log("HOMER", data);
           const handler = Plaid.create({{
-            env: {PLAID_ENV},
+            env: '{PLAID_ENV}',
```
Comment (andrewbuccilli) [inline app/handlers.py:301]: added quotes

File: app/handlers.py [hunk 5]
```diff
@@ -188,7 +187,7 @@ def plaid_bond_test(account_id, PLAID_ENV):
 
       function initializePlaidLink( data ) {{
         const handler = Plaid.create({{
-          env: {PLAID_ENV},
+          env: '{PLAID_ENV}',
```
Comment (andrewbuccilli) [inline app/handlers.py:190]: added quotes

**Review by adityabond — APPROVED** [review]

File: app/handlers.py (hunk 5 above)
Comment (adityabond) [reply app/handlers.py:190]: This will already be there, right?

**Comment by adityabond** [comment]
Thanks for the changes!

File: app/handlers.py (hunk 5 above)
Comment (andrewbuccilli) [reply app/handlers.py:190]: the whole thing is a string, need explicit quotes around this string, else getting:

```
52 Uncaught (in promise) ReferenceError: sandbox is not defined
    at initializePlaidLink (52)
    at 92
```

Comment (adityabond) [reply app/handlers.py:190]: Thanks for confirming

**Pull request merged**